.travis.yml

```diff
@@ -1,6 +1,6 @@
 language: node_js
 node_js:
-- '5'
+- '6.10.3'
```

Comment:
I disagree, and in fact would like us to get all our packages updated, including node 8. Immerse also runs fine with Node 6. https://nodejs.org/en/blog/release/v6.0.0/

What is the process by which we resolve this?